gamemodes/horde/gamemode/init.lua

```diff
@@ -145,10 +145,16 @@ AddCSLuaFile("arccw/attachments/horde_ubgl_medic.lua")
 AddCSLuaFile("arccw/attachments/horde_ubgl_m203.lua")
 AddCSLuaFile("arccw/attachments/horde_ammo_ap.lua")
 AddCSLuaFile("arccw/attachments/horde_ammo_sabot.lua")
+AddCSLuaFile("arccw/attachments/bo1_skorpion_ext_mag.lua")
```

Comment:
these are addcsluafile including it twice, this can probably be removed

Reply:
just spent the afternoon testing this and it seems that the current implementation is the least buggy/problematic